jiratools/example_config.py

```diff
@@ -0,0 +1,29 @@
+"""Build Example Config."""
+import argparse
+import os
+import shutil
+import sys
+
+from jgt_common import error_if
+
+from .utils import CONFIG_FILENAME, SAMPLE_CONFIG_FILENAME
+
+
+def cli_example_config(args: argparse.Namespace) -> None:
+    """Build example config."""
+    error_if(
+        not os.path.exists(SAMPLE_CONFIG_FILENAME),
+        message="Missing example config file: {}".format(SAMPLE_CONFIG_FILENAME),
+    )
+
+    prefix = "Copying" if args.install else "Would copy"
+    print("{} {} to {}".format(prefix, SAMPLE_CONFIG_FILENAME, CONFIG_FILENAME))
+    if args.install:
+        message = "Not installing, config file already in place: {}".format(
+            CONFIG_FILENAME
```

Comment:
aha! here is where the travis travesity of working around the `--install` error happens.
Having thought about it for all of 10 minutes, I think having a default dry-run is the right thing, having a 'install or fail' is good for human use, _and_ having an 'install if needed' option is "just-right" for CICD situations. (In most cases the file will exist, but if the server is reset, a new server is started up, etc. it'll "just work").

Reply:
I think this is where some additional discussion may be appropriate?